tests/dummy/app/components/cookbook/client-side-table.js

```diff
@@ -57,55 +62,56 @@ export default BaseTable.extend({
         valuePath: 'country',
       },
     ];
-  }),
+  }
 
-  fetchRecords: task(function* () {
-    let records = yield this.store.query('user', { page: 1, limit: 100 });
+  @task({ on: 'init' }) *fetchRecords() {
```

Comment:
@maxwondercorn would it be possible to call this in `init` instead of doing `on: 'init'`?

Reply:
Yes, more clear code.  I was just refactoring and didn't catch this.  Fix pushed